internal/extractors/instruments.go

```diff
@@ -57,3 +57,29 @@ func instrumentTransparentData(rid aez.RID, buf io.Reader) (common.Exportable, e
 	}
 	return dataPackage, err
 }
+
+func instrumentVerification(subtype uint8, buf io.Reader) (common.Exportable, error) {
+	var dataPackage common.Exportable
+	var err error
+	switch subtype {
+	case 1: // TC Acceptance Report - Success

```

Comment:
I think I'd prefer these as const of a dedicated type defined in `aez/verification.go` like we do for SID and RID